docs/README.md

```diff
@@ -103,6 +103,13 @@ Handles subtitle requests.
 
 [Subtitle Request Parameters and Example](./api/requests/defineSubtitlesHandler.md)
 
+
+#### `builder.defineResourceHandler('addon_catalog', function handler(args) { })`
+
+Handles add-on catalog requests, this can be used by an add-on to just send a list of other add-on manifests.
```

Comment:
we've decided to spell it addon: https://github.com/stremio/smartcode-org#terminology

Reply:
Yes, I know, but unless I would of changed the entirety of the docs, then naming them "addons" just for these changes would of led to inconsistencies in spelling.

Reply:
IMO, changing how addons are spelled should be a separate PR, so we keep the commits clean and readable.